apps/framework-docs/src/pages/getting-started/moose-cli.mdx

```diff
@@ -0,0 +1,37 @@
+# Moose CLI
```

Comment:
I was looking at the preview and I am not sure how to get to this page.

If this was the link on the left "Using the CLI" the link is broken. I left a comment on vercel as well.

Reply:
the link is called "Moose CLI" and it works, as far as i can tell. can you try again and confirm the issue here?

Reply:
i think its fixed. going to redeloy we'll see.